src/main/java/edu/phema/elm_to_omop/translate/criteria/correlation/CorrelatedQuickPair.java

```diff
@@ -0,0 +1,60 @@
+package edu.phema.elm_to_omop.translate.criteria.correlation;
+
+/**
+ * Class representing two QuickResourceAttributePairs, used to represent correlations that we support
+ */
+public class CorrelatedQuickPair {
+    private QuickResourceAttributePair outerResourceAttributePair;
+    private QuickResourceAttributePair innerResourceAttributePair;
+
+    /**
+     * Create from existing QuickResource and QuickResourceAttribute objects
+     *
+     * @param outerQuickResource          The outer QUICK resource
+     * @param outerQuickResourceAttribute The outer QUICK resource attribute
+     * @param innerQuickResource          The outer QUICK resource
+     * @param innerQuickResourceAttribute The outer QUICK resource attribute
+     * @throws CorrelationException
+     */
+    public CorrelatedQuickPair(QuickResource outerQuickResource, CorrelationConstants.QuickResourceAttribute outerQuickResourceAttribute,
+                               QuickResource innerQuickResource, CorrelationConstants.QuickResourceAttribute innerQuickResourceAttribute) throws CorrelationException {
+
+        outerResourceAttributePair = new QuickResourceAttributePair(outerQuickResource, outerQuickResourceAttribute);
+        innerResourceAttributePair = new QuickResourceAttributePair(innerQuickResource, innerQuickResourceAttribute);
+    }
+
+    /**
+     * Create from a QUICK resources type name and QuickResourceAttribute objects
+     *
+     * @param outerQuickResourceName      The outer QUICK resource type name (e.g. "Encounter")
+     * @param outerQuickResourceAttribute The outer QUICK resource attribute
+     * @param innerQuickResourceName      The outer QUICK resource type name
```

Comment:
Same as above, minor consistency in comment to change "outer" to "inner" on L31-32.